File: Assignment_49/Q2.java
///////////////////////////////////////////////////////////////////////////////////////////////
//
//  File name :     Q1.java
//  Description : write java program which accept string from user 
//                  and display below pattern.
//  Author :        Vaishanvi D Shingare
//  Date :          2/6/2025

/* INPUT  : Hello
   OUTPUT : H e l l o
            H e l l *
            H e l * * 
            H e * * * 
            H * * * * 
*/
           
///////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Q2
{
        public void Pattern(String str) 
    {
        int ino = str.length();
        int i = 0 ,j = 0;

        for ( i = 0 ; i < ino ; i++) 
        {
            for ( j = 0; j < ino ; j++) 
            {
                if(j < (ino - i))
                {
                    System.out.print(str.charAt(j) + " ");
                }
                else
                {
                    System.out.print("* ");

                }
            }
            System.out.println();
        }

    }

    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String input = sobj.nextLine();

        Q2 obj = new Q2();
        obj.Pattern(input);
    }
}

File: Assignment_40/Q2.c
///////////////////////////////////////////////////////
//
//  File name :     assignment40-2.c
//  Descreption :   write a recursive program which display below pattern.
//  Author :        Vaishanvi D Shingare
//  Date :          28/04/2025
//
//  output  :   1  2  3  4  5
///////////////////////////////////////////////////////

void Display()
{
    static int i = 1;

    while(i < 6)
    {
        printf("%d \t",i);
        i++;
    }

}

int main()
{
    Display();

    return 0;
}